client-lite/src/ipc/rest_api_request.cpp

```diff
@@ -134,7 +134,6 @@ HRESULT RestApiDownloadStateChangeRequest::ParseAndProcess(DownloadManager& down
 HRESULT RestApiGetStatusRequest::ParseAndProcess(DownloadManager& downloadManager, RestApiParser& parser,
     web::json::value& responseBody)
 {
-    DoLogInfo("");
```

Comment:
nice catch, this was doing nothing before?

Reply:
I think the intention was to log the entry to this method, which it does. However, gcc only provides the method name and not the class name so it is not so useful here.